dspace-api/src/main/java/org/dspace/scripts/DSpaceRunnable.java

```diff
@@ -24,8 +31,9 @@
  * it provides the basic variables to be hold by the script as well as the means to initialize, parse and run the script
  * Every DSpaceRunnable that is implemented in this way should be defined in the scripts.xml config file as a bean
  */
-public abstract class DSpaceRunnable implements Runnable {
+public abstract class DSpaceRunnable implements Runnable, BeanNameAware {
 
+    private UUID epersonIdentifier;
```

Comment:
if this attribute hold information related to a specific execution this imply that the bean needs to be declared as `prototype` or we need to force a single thread model. Both options are suboptimal, am I misreading?